experiments/node-cpu-hog/experiment_cron.yaml

```diff
@@ -76,7 +76,7 @@ spec:
                           verbs:
                             - "get"
                             - "list"
-                      image: "litmuschaos/go-runner:latest"
+                      image: "litmuschaos/go-runner:3.0.0-beta10"
```

Comment:
Can we keep the `latest` image tag in the master branch? otherwise, we need to keep updating it for every release